bump.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Usage:
+ * Change directory to project folder
+ * node /path/to/script versionNumber
+ */
+const fs = require('fs');
+const encoding = 'utf-8';
+
+// File Location
+const updateVersions = [
+    mdFile =  {
+        "path" : "./README.md",
+        "pattern": /\d{1,}\.\d{1,}\.\d{1,}/
+    },
+    packageFile = {
+        "path" : "./package.json",
+        "pattern": /\d{1,}\.\d{1,}\.\d{1,}/
+    }
+]
+
+// Require a version to update
+if (process.argv[2] === undefined){
+    throw new Error("Missing version string for update");
+}
+
+for (let i = 0; i < updateVersions.length; i++){
```

Comment:
Nitpick: While any loop will work for this, a for loop is better used for more special cases like iterating over a loop backwards, or maybe looping at only odd indexes, etc... A better approach would be to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

No need to change this, just an FYI